pkg/checks/external/main.go

```diff
@@ -1026,9 +1026,40 @@ func (ext *Checker) createPod() (*apiv1.Pod, error) {
 	// enforce various labels and annotations on all checker pods created
 	ext.addKuberhealthyLabels(p)
 
+	// Get the hostname of the kuberhealthy pod
+	kPodName, err := os.Hostname()
+	if err != nil {
+		ext.log("Unable to get kuberhealthy pod podname")
+	}
+
+	// Get the podspec of kuberhealthy pod
+	kHealthyPod, err := ext.getKuberHealthyPod(kPodName)
+	if err != nil {
+		ext.log("Unable to find kuberhealthy pod", err)
+	} else {
+		// Sets OwnerRefernces on all checker pods
+		p.OwnerReferences = []metav1.OwnerReference{
+			{
+				APIVersion: "v1",
+				Kind:       "Pod",
```

Comment:
I tried to find a way to not hardcode the APIVersion and Kind but I was unable to do so.
I'm sure there is a nice way through the k8s core api, if you know it please point it out.

Reply:
Tried with apiv1.Pod{}.APIVersion but this just returns an empty string